libs/ledger-live-common/src/families/ethereum/specs.ts

```diff
@@ -131,7 +131,7 @@ const ethereum: AppSpec<Transaction> = {
   appQuery: {
     model: DeviceModelId.nanoS,
     appName: "Ethereum",
-    appVersion: "1.9.20-dev", // FIXME remove this line once 1.9.20 lands on coin-apps (branch ledger-live-bot)
+    appVersion: "1.9.20",
```

Comment:
@lambertkevin Any reason for not removing the line?
This works but it will prevent all bots from doing non-reg on new versions of the Eth Nano app, which is a small bonus we have by default.